spec/features/translations_spec.rb

```diff
@@ -71,6 +71,18 @@
       expect(page).not_to have_link "Español"
     end
 
+    scenario 'Change value of a translated field to blank' do
+      milestone.update_attributes!(status: create(:budget_investment_status))
```

Comment:
IMHO this line isn't necessary.

Reply:
You're right, I can't remember why I put that there... the test passes without it. I'll remove it.

Reply:
Removed